scripts/generate_unaccent_rules.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+"""
+***************************************************************************
+    generate_unaccent_rules.py
+    ---------------------
+    Date                 : November 2025
+    Copyright            : (C) 2025 by Tudor Bărăscu
+    Email                : tudor dot barascu at qtibia dot ro
+***************************************************************************
+*                                                                         *
+*   This program is free software; you can redistribute it and/or modify  *
+*   it under the terms of the GNU General Public License as published by  *
+*   the Free Software Foundation; either version 2 of the License, or     *
+*   (at your option) any later version.                                   *
+*                                                                         *
+***************************************************************************
+"""
+
+__author__ = "Tudor Bărăscu"
+__date__ = "November 2025"
+__copyright__ = "(C) 2025, Tudor Bărăscu"
+
+from pathlib import Path
+import re
+import sys
+
+
+ROOT = Path(__file__).resolve().parent.parent
+RULES_FILE = ROOT / "resources" / "unaccent.rules"
+OUT_FILE = ROOT / "src" / "core" / "qgsunaccent_generated_rules.cpp"
```

Comment:
Just a minor comment. Filename is a bit inconsistent, the rest of the QGIS source files don't have underscores in the names. Should we keep it as is or remove underscores to match the rest of the codebase?